_posts/2015-09-14-discover-and-communicate-with-nearby-bluetooth-devices.markdown

```diff
@@ -0,0 +1,151 @@
+---
+layout: post
+title:  "Discover and communicate with nearby Bluetooth devices"
+date:   2015-09-14 9:32:55
+author: fbeaufort
+categories: announcements
+excerpt: A new set of Polymer elements that simplify discovering and communicating with nearby Bluetooth devices, powered behind the scenes by the experimental Web Bluetooth API is now available.
+
+---
+
+I'm particularly glad to share with you today the [platinum bluetooth elements](https://elements.polymer-project.org/elements/platinum-bluetooth?active=platinum-bluetooth-device), a
+brand new set of [Polymer platinum
+elements](https://elements.polymer-project.org/browse?package=platinum-elements)
+to discover and communicate with nearby Bluetooth devices powered behind the
```

Comment:
Might make this two sentences - e.g. "...to dicsover and communicate with nearby Bluetooth devices. The new components are powered behind the scenes by the experimental..."